pcl_hline_reducer:
  sub_topic: "/os_cloud_node/points"
  pub_topic: "/pcl_hline_reduced"
  pcl_height: 128
  pcl_width: 2048
  pcl_target_height: 64

pizza_filter:
  sub_topic: "/os_cloud_node/points"
  pub_topic: "/pcl_hline_pizzad"
  pcl_height: 128
  pcl_width: 2048
  starting_angle: 0.0
  pizza_width: 45.0